servicex/templates/app/configmap.yaml

```diff
@@ -103,23 +103,32 @@ data:
 
     {{ if .Values.objectStore.enabled }}
     OBJECT_STORE_ENABLED = True
+      {{ if not .Values.objectStore.internal }}
+    # using external minio 
+    MINIO_URL = '{{ .Values.objectStore.publicURL }}'
+    MINIO_URL_TRANSFORMER = '{{ .Values.objectStore.publicURL }}'
+    MINIO_PUBLIC_URL = '{{ .Values.objectStore.publicURL }}'
+    MINIO_SECURED = {{ ternary "True" "False" .Values.objectStore.usetls }}
```

Comment:
This is eventually a python config file. You should remove the quotes around `True` and `False`

Reply:
Without the quotes, helm looks for a function called True and errors out.  The configmap gets the strings without the quotes.

Reply:
Also updated the docs with the new settings

Reply:
I don't see any changes to the [deployment reference docs](https://github.com/ssl-hep/ServiceX/blob/b28498a96a5df99f57ac0a261ea679175f02cf70/docs/deployment/reference.md?plain=1#L8)

Reply:
Pushed an update